src/GeneralStateTestsFiller/stCreateTest/CreateOOGafterInitCodeReturndataFiller.json

```diff
@@ -0,0 +1,68 @@
+{
+    "CreateOOGafterInitCodeReturndata" : {
+	"_info" :{
+	    "comment" : "Call RETURNDATASIZE and RETURNDATACOPY after CREATE deploy a contract"
+	},
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x020000",
+            "currentGasLimit" : "10000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : -1,
+                    "value" : -1
+                },
+                "network" : [">=Byzantium"],
+                "result" : {
+		    "b94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+		        "storage" : {
+		            "0x01" : "0"
+		        }
+		    },
+		    "0xf1ecf98489fa9ed60a664fc4998db699cfa39d40" : {
+		        "shouldnotexist" : "1"
+		    }
+                }
+            }
+        ],
+        "pre" :
+        {
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	    "b94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "0",
+		"nonce" : "0",
+		"//code" : "(SSTORE 1 1) RETURN",
+		"code" : "{ (MSTORE 0 0x6460016001556000526005601bf3) (CREATE 0 18 14) [[ 1 ]] (RETURNDATASIZE) (RETURNDATACOPY 0 0 32) [[ 2 ]] (MLOAD 0) }",
```

Comment:
This is wrong, because the `RETURNDATACOPY` will cause the failure. You are not allowed to access more that at was returned and `CREATE` returns nothing.

Reply:
I know.